src/main/java/io/cify/common/Constants.java

```diff
@@ -0,0 +1,7 @@
+package io.cify.common;
+
+public final class Constants {
+    public static final String CIFY_CONFIG_FILE_NAME = "capabilities.json";
```

Comment:
It limits "capabilities.json" to be only in the root folder or any other fixed (and unclear) place. Should be configurable or searchable